web/server/entrypoint.go

```diff
@@ -18,21 +19,34 @@ import (
 	"go.viam.com/utils/perf"
 	"go.viam.com/utils/rpc"
 
+	"go.viam.com/rdk/components/camera/videosource/logging"
 	"go.viam.com/rdk/config"
 	robotimpl "go.viam.com/rdk/robot/impl"
 	"go.viam.com/rdk/robot/web"
 	weboptions "go.viam.com/rdk/robot/web/options"
 	rutils "go.viam.com/rdk/utils"
 )
 
-var viamDotDir = filepath.Join(os.Getenv("HOME"), ".viam")
+var (
+	// GLogger is the global logger-to-file.
+	GLogger    *logging.Logger
```

Comment:
[nit] I would name this so that it's clear this is the camera logger.

Reply:
Done